src/components/views/settings/tabs/user/PreferencesUserSettingsTab.tsx

```diff
@@ -23,10 +23,24 @@ import Field from "../../../elements/Field";
 import * as sdk from "../../../../..";
 import PlatformPeg from "../../../../../PlatformPeg";
 import {SettingLevel} from "../../../../../settings/SettingLevel";
-import {replaceableComponent} from "../../../../../utils/replaceableComponent";
+import { replaceableComponent } from "../../../../../utils/replaceableComponent";
+
+interface IState {
+    autoLaunch: boolean;
+    autoLaunchSupported: boolean;
+    warnBeforeExit: boolean;
+    warnBeforeExitSupported: boolean;
+    alwaysShowMenuBarSupported: boolean;
+    alwaysShowMenuBar: boolean;
+    minimizeToTraySupported: boolean;
+    minimizeToTray: boolean;
+    autocompleteDelay: string,
+    readMarkerInViewThresholdMs: string,
+    readMarkerOutOfViewThresholdMs: string,
```

Comment:
we should probably agree on `;` vs `,` - we seem to have more `;`s, you seem to mix them

Reply:
Ah just a mistake, will fix to `;`. Probably we should lint it...